src/Glpi/Http/GlpiPageResponse.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Http;
+
+use Glpi\Application\View\TemplateRenderer;
+use Symfony\Component\HttpFoundation\Response;
+
+final class GlpiPageResponse extends Response
+{
+    public function __construct(
+        string $view,
+        array $parameters = [],
+        int $status = 200,
+        array $headers = [],
+    ) {
+        $twig = TemplateRenderer::getInstance();
+
+        // We must use output buffering here as Html::header() and Html::footer()
+        // output content directly.
+        // TODO: fix header() and footer() methods and remove output buffering
```

Comment:
The real solution would be to rewrite the whole header and footer logic to render them directly in the skeleton. It cannot be done now, I add it to the technical backlog.